src/proctored-exam-settings/ProctoredExamSettings.jsx

```diff
@@ -81,100 +81,136 @@ function ExamSettings(props) {
   }
 
   function getProctoringProviderOptions(providers) {
-    return providers.reduce(
-      (accumulator, currentValue) => {
-        accumulator.push({
-          value: currentValue,
-          label: currentValue,
-          disabled: isDisabledOption(currentValue),
-        }); return accumulator;
-      }, [],
-    );
+    return providers.map(provider => (
+      <option
+        key={provider}
+        value={provider}
+        disabled={isDisabledOption(provider)}
+      >
+        {provider}
+      </option>
+    ));
   }
 
   function renderContent() {
     return (
-      <div className="content-container">
-        <ValidationFormGroup
-          for="enableProctoredExams"
-          helpText="If checked, proctored exams are enabled in your course."
-        >
-          <CheckBox
+      <Form>
+        {proctortrackEscalationEmailError
```

Comment:
I moved this alert to the top of the form. Form validation errors should be shown at the top of the form for accessibility reasons. This allows the user to go to the top of the form, see the validation errors, and go back through the form to change the errors.